.github/workflows/unit-tests.yaml

```
@@ -1,14 +1,29 @@
 name: Unit Tests
-on:
+on: 
+  push:
+    branches:
+      - incubation
+      - rhoai
+      - main
+      - master
```

Just FYI this is enough as the workflow is taken from the associated commit. In this case it will be from the `rhoai` branch.
```suggestion
      - rhoai
```